pages/render.js

```diff
@@ -9,15 +9,17 @@ export default function RenderPage() {
   return (
     <>
       <HeaderResponsive type="B2C" />
-      <Container>
-        <Row>
-          <Col>
-            <Header title="Simulates render of one component" />
-            <p>Here you implement a component to make all use tests</p>
-          </Col>
-        </Row>
-      </Container>
-      <FooterTagsList />
+      <section>
+        <Container>
+          <Row>
+            <Col>
+              <Header title="Simulates render of one component" />
+              <p>Here you implement a component to make all use tests</p>
+            </Col>
+          </Row>
+        </Container>
+        <FooterTagsList />
+      </section>
```

Comment:
```suggestion
      <section>
        <Container>
          <Row>
            <Col>
              <Header title="Simulates render of one component" />
              <p>Here you implement a component to make all use tests</p>
            </Col>
          </Row>
        </Container>
      </section>
      <FooterTagsList />
```
When thinking of the "render" page as an independent page, the footer used here has information about the entire site page and not just information pertaining to one section of the page. In this case, I believe that the section should not have the footer as a child, just the container. More details in the footer document https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer#usage_notes